src/app/business/delfin/masking-views/masking-views.component.ts

```diff
@@ -0,0 +1,146 @@
+import { Component, OnInit, ViewContainerRef, ViewChild, Directive, ElementRef, HostBinding, HostListener, PipeTransform, Pipe, Input,  } from '@angular/core';
+import { DomSanitizer } from "@angular/platform-browser";
+import { Router, ActivatedRoute } from '@angular/router';
+import { I18NService, Utils, Consts} from 'app/shared/api';
+import { FormControl, FormGroup, FormBuilder, Validators, ValidatorFn, AbstractControl } from '@angular/forms';
+import { AppService } from 'app/app.service';
+import { I18nPluralPipe } from '@angular/common';
+import { trigger, state, style, transition, animate } from '@angular/animations';
+import { Message, MenuItem ,ConfirmationService} from '../../../components/common/api';
+import { DelfinService } from '../delfin.service';
+
+
+
+let _ = require("underscore");
+
+
+@Component({
+    selector: "app-delfin-masking-views",
+    templateUrl: "masking-views.component.html",
+    providers: [ConfirmationService],
+    styleUrls: [],
+    animations: [],
+})
+export class MaskingViewsComponent implements OnInit {
+    @Input() selectedStorage;
+
+    selectedStorageId: any;
+    showRightSidebar: boolean = false;
+    showDetails: boolean = false;
+    selectedMaskingView: any;
+    allMaskingViews: any;
+    msgs: Message[];
+    detailsLabels = {
+        heading: ""
+    }
+    detailSource: any;
+    detailsObj: any;
+    label = {
+        name: "Name",
+        description: "Description",
+        createdAt: "Created At",
+        updatedAt: "Updated At",
+        hostName: "Name",
+        ip: "IP Address",
+        port: "Port",
+        status: "Status",
+        osType: "OS",
+        accessMode: "Access Mode",
+        availabilityZones: "Availability Zones",
+        initiators: "Initiators",
+        storage_id: "Storage ID",
+        native_storage_host_id: "Native Storage Host ID"
+  };
+
+    constructor(
+        public i18n: I18NService,
+        public ds: DelfinService,
+        private confirmationService: ConfirmationService,
+        private fb: FormBuilder,
+        private router: Router,
+        private ActivatedRoute: ActivatedRoute
+    ) {}
+
+    ngOnInit() {
+        this.selectedStorageId = this.selectedStorage;
+        this.getAllMaskingViews(this.selectedStorage);
+    }
+
+    getAllMaskingViews(storageId){
+        this.ds.getAllMaskingViews(storageId).subscribe((res)=>{
+            let allViews = res.json().masking_views
+            
+            
+            allViews.forEach(view => {
+                view['storageHostInitiatorsCount'] = 0;
+                view['volumesCount'] = 0;
```

Comment:
Is this displayed somewhere?

Reply:
Not at the moment. The count was maintained earlier to show in the column.